recipes/cupy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "cupy" %}
+{% set version = "6.4.0" %}
+{% set sha256 = "2a3cd1812f043ba9451b8ca4e5a4c31f9acc486c003f6970836e1a46c735c82e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cupy/cupy/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not (linux64 and cuda_compiler_version != "None")]
```

Comment:
Using De Morgan's Laws, this skip statement could be made easier to read:
```
[(not linux64) or cuda_compiler_version == "None"]
```

Reply:
A PR to the feedstock would be welcome. 🙂

ref: https://github.com/conda-forge/cupy-feedstock

Reply:
:sunglasses:

Reply:
xref: https://github.com/conda-forge/cupy-feedstock/pull/4